content/platforms/release-notes/k8s-6-0-6-23-2020-08.md

```diff
@@ -23,6 +23,13 @@ This release of the operator provides:
 
 Follow the instructions in ["Upgrading a Redis Enterprise Cluster in Operator-based Architecture"]({{< relref "/platforms/kubernetes/upgrading-kubernetes-with-operator.md" >}}) to upgrade to this Kubernetes operator release.
 
+{{< note >}}
+
+When upgrading the operator to 6.0.6-23 with cluster deployments, the StatefulSet for the Redis Enterprise cluster has minor
+updates that will cause a rolling update of the pods even though the version of Redis Enterprise stays the same (6.0.6-39).
```

Comment:
This line doesn't make sense to me so maybe read it again and confirm.
If the operator is upgraded to 6.0.6-23 and the pods are updated (to what version?) how is it that the RE version is ahead at 6.0.6-23?

Reply:
The operator updates the pods inside K8s causing the rolling update. There is no *version* update but metadata update to the deployment. We aren't talking about versions of software.

Reply:
This should have been committing awhile ago.